azure/services/resourceskus/sku.go

```diff
@@ -128,3 +130,23 @@ func (s SKU) GetCapability(name string) (string, bool) {
 	}
 	return "", false
 }
+
+// HasLocationCapability returns true if the provided resource supports the location capability.
+func (s SKU) HasLocationCapability(name, size, location string) bool {
+	if s.LocationInfo != nil && *s.Name == size {
```

Comment:
the check on sku.name shouldn't be necessary, since we fetch it from the cache by name already